src/store/reviews-load/reviews-load.ts

```diff
@@ -14,7 +15,20 @@ export const reviewsLoad = createSlice({
   extraReducers(builder) {
     builder
       .addCase(fetchReviewsAction.fulfilled, (state, action) => {
+        state.isReviewFormDasabled = false;
         state.reviews = action.payload;
+      })
+      .addCase(fetchReviewsAction.rejected, (state) => {
+        state.isReviewFormDasabled = false;
```

Comment:
Есть подозрение, что либо ты не всё запушил, либо пропустил часть комментариев

```suggestion
        state.isReviewFormDisabled = false;
```